torch/fx/proxy.py

```diff
@@ -261,6 +269,47 @@ def node(self):
     def __call__(self, *args, **kwargs):
         return self.tracer.create_proxy('call_method', self.attr, (self.root,) + args, kwargs)
 
+
+class ParameterProxy(Proxy):
+    """
+    a special proxy which lets "shape", "size", "dim", and a few other
+    attribute accesses pass through to the underlying  module parameter object,
+    so that conditional tests on these attributes will not throw exception during tracing
+    """
+    def __init__(self, tracer: TracerBase, node: Node, name, param):
+        super().__init__(node, tracer)
+        assert(isinstance(param, torch.nn.Parameter))
+        self.param = param
+        self.name = name
+
+    def __repr__(self) -> str:
+        return f'ParameterProxy({self.name})'
+
+    @property
+    def shape(self):
+        return self.param.shape
+
+    @property
+    def size(self):
+        return self.param.size
+
+    @property
+    def dim(self):
+        return self.param.dim
+
+    @property
+    def ndim(self):
+        return self.param.ndim
+
+    @property
+    def numel(self):
+        return self.param.numel
+
+    @property
+    def nelement(self):
+        return self.param.nelement
```

Comment:
All of these except for `ndim` are methods, not properties. Please add tests!

Reply:
added tests. Now 6 tests in total for each field.